tests/driver_lis2dh12/main.c

```diff
@@ -26,6 +26,7 @@
 #include "mutex.h"
 #include "lis2dh12.h"
 #include "lis2dh12_params.h"
+#include "lis2dh12_registers.h"
```

Comment:
Did you forget to commit the remaining lines of this file? :wink: 
It would be good to have a test for the new functionality.